### ISSUE TITLE
Glitch embedded app not working

### ISSUE THREAD

**Issue by yysun** [thepracticaldev/dev.to#6589]
I tried the glitch app embedding in the post. https://dev.to/yysun/database-driven-applications-using-websockets-2b9o

{% glitch apprun-websockets-sqlite app %}

The embedded app displays correctly in the preview.  It displays the default glitch template, not my app when not signed in. Nor in the mobile app.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @yysun, thanks for raising the issue. 

I just tried with both 

`{% glitch apprun-websockets-sqlite %}`

and

`{% glitch apprun-websockets-sqlite app %}`

and it worked but they were the only embeds in the page:

![Screenshot_2020-03-11 test glitch embed - DEV(local) Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/76435443-48d05800-63b7-11ea-94a8-2fe022fea44f.png)

![Screenshot_2020-03-11 test glitch embed - DEV(local) Community 👩‍💻👨‍💻(1)](https://user-images.githubusercontent.com/146201/76435463-4d950c00-63b7-11ea-9a08-d720d1ac6fec.png)

I can also see the embed in your article. It just takes quite a lot of time to show. 

![Screenshot_2020-03-11 Database-Driven Applications Using WebSockets - DEV Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/76435980-f9d6f280-63b7-11ea-9c68-175c9e70e0e9.png)

Apparently Glitch itself is having some issues, see https://status.glitch.com/ :

![Screenshot_2020-03-11 Glitch Status](https://user-images.githubusercontent.com/146201/76436051-0f4c1c80-63b8-11ea-9d02-95d23a874446.png)

I'm going to close this in the meantime. Thanks a lot for letting us know!

I think it's just a matter of waiting for those to resolve as we can't do anything on our part.

**Comment by yysun** [comment]
Thank you so much for such a quick response. 

You actually replicated the issue. My app on glitch is a Todo app like the one in the embedded repl.it.  "A Dream of the Future" is not my app. What I see in the editing preview is different to what you see.

![image](https://user-images.githubusercontent.com/170547/76437198-933dde80-638f-11ea-9f44-4721226cdbc3.png)

**Comment by mandaputtra** [comment]
Just tried it works

![Screenshot from 2020-03-12 09-43-02](https://user-images.githubusercontent.com/23342943/76482159-f25e2a80-6445-11ea-99bb-c19b27ce1acf.png)

**Comment by rhymes** [comment]
I think it's working now @yysun, I'm going to close this. Let me know if that's not the case

**Comment by yysun** [comment]
I am sorry, but It is NOT working.
Expected:
![](https://user-images.githubusercontent.com/170547/76437198-933dde80-638f-11ea-9f44-4721226cdbc3.png)

**Comment by rhymes** [comment]
hi @yysun I might not have understood what the issue is though. What is supposed to happen? It seems like the glitch is loading

**Comment by yysun** [comment]
Glitch shows but what it shows is not is not my project.

**Comment by maestromac** [comment]
Hello @yysun , the reason you are experiencing this error is that the glitch id you've provided either no longer exist, invalid, or private (assuming it's  http://apprun-websockets-sqlite.glitch.me). 

I've played around with glitch embed directly on glitch, and what I found is that when the invalid id is provided, Glitch will use `window.alert` to inform you of the expected error and render a random Glitch content. Unfortunately, we don't have a similar form of error handling as it will be disruptive to our users.

I hope that clear thing up.
